packages/synapse-types/src/FeatureFlags.ts

```diff
@@ -15,4 +15,7 @@ export enum FeatureFlagEnum {
 
   // If enabled, show the chatbot entrypoint from the new Synapse Homepage
   HOMEPAGE_CHATBOT = 'HOMEPAGE_CHATBOT',
+
+  // If enabled, show the Portals dropdown on the portal site (to discover other portals)
+  PORTALS_DROPDOWN = 'PORTALS_DROPDOWN',
```

Comment:
Please also make sure this is added to the stack builder in https://github.com/Sage-Bionetworks/Synapse-Stack-Builder/blob/develop/src/main/resources/templates/repo/appconfig-config.json

Reply:
https://github.com/Sage-Bionetworks/Synapse-Stack-Builder/pull/662